### DISCUSSION TITLE
How to define a custom function to be executed in a workflow

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15552] [How to] [answered]
I would like to execute a custom function (in php) that I have written each time a record is modified. I saw that it is possible to create a workflows that allows executing a custom function.
![xxx](https://user-images.githubusercontent.com/10230144/146355748-b799aad1-0c13-4f9d-b5e8-8acdb737255b.jpg)

Where do I have to define such functions in order for them to be selectable (and what is their specific form, i.e., where can I find an example of such a custom function / a template).
![yyy](https://user-images.githubusercontent.com/10230144/146355855-ba09907b-96b2-4710-bc7b-e328de627d9c.jpg)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Table: com_vtiger_workflowtasks_entitymethod
![image](https://user-images.githubusercontent.com/10198654/146356391-afaf1635-51ed-47e4-8bd8-9b455937056b.png)
